docs/self-managed/platform-deployment/helm-kubernetes/platforms/amazon-eks/eksctl.md

```diff
@@ -0,0 +1,615 @@
+---
+id: eks-eksctl
+title: "Deploy an AWS Kubernetes Cluster (EKS) with eksctl"
+description: "Deploy Camunda 8 Self-Managed on Amazon EKS with eksctl."
+---
+
+This guide explores the streamlined process of deploying Camunda 8 Self-Managed on Amazon Elastic Kubernetes Service (EKS) using the `eksctl` command-line tool.
+
+[Eksctl](https://eksctl.io/) is a common CLI tool for quickly creating and managing your AWS EKS clusters and is [officially endorsed](https://docs.aws.amazon.com/eks/latest/userguide/getting-started-eksctl.html) by Amazon.
+
+This guide provides a user-friendly approach for setting up and managing AWS EKS clusters. It covers everything from the prerequisites, such as AWS IAM role configuration, to creating a fully functional AWS EKS cluster and a managed Aurora PostgreSQL instance. Ideal for those seeking a practical and efficient method to deploy Camunda 8 on AWS, this guide provides detailed instructions for setting up the necessary environment and AWS IAM configurations.
+
+## Prerequisites
+
+- An [AWS account](https://docs.aws.amazon.com/accounts/latest/reference/accounts-welcome.html) is required to create resources within AWS.
+- [AWS CLI (2.11+)](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html), a CLI tool for creating AWS resources.
+- [eksctl (0.163+)](https://eksctl.io/installation/), a CLI tool for creating and managing EKS clusters.
+- [kubectl (1.28+)](https://kubernetes.io/docs/tasks/tools/#kubectl), a CLI tool to interact with the cluster.
+
+## Considerations
+
+This is a basic setup to get started with Camunda 8 but does not reflect a high performance setup. For a better starting point towards production, we recommend utilizing [Infrastructure as Code tooling](https://developer.hashicorp.com/terraform/tutorials/aws-get-started/infrastructure-as-code) and having a look at our [Terraform guide](./terraform-setup.md).
+
+To try out Camunda 8 or develop against it, consider having a look at our [SaaS offering](https://camunda.com/platform/) or if you already have an AWS EKS cluster, consider skipping to the [Helm guide](./eks-helm.md).
+
+While the guide is primarily tailored for UNIX systems, it can also be run under Windows by utilizing the [Windows Subsystem for Linux](https://learn.microsoft.com/windows/wsl/about).
+
+:::warning
+Following this guide will incur costs on your Cloud provider account.
+:::
+
+<!--- Can you elaborate? What kinds of costs and why? --->
```

Comment:
And here 👍

Reply:
Elaborated on what is causing costs and linked to AWS Docs and the pricing calculator as it will differ per region.

Reply:
Awesome 🚀